src/main/java/mekanism/common/integration/multipart/MultipartMekanism.java

```diff
@@ -162,4 +164,28 @@ public static boolean hasConnectionWith(TileEntity tile, EnumFacing side)
 		
 		return Collections.emptyList();
 	}
+	
+	public static boolean isMultiPartContainer(TileEntity entity)
+	{
+		return entity instanceof IMultipartContainer;
+	}
+	
+	public static Collection<TileEntity> getMekanismTileEntities(TileEntity container)
+	{
+		ArrayList<TileEntity> entities = new ArrayList<>();
```

Comment:
I don't see why this needs to return a list; there will only ever be one TileEntitySidedPipe in the block as far as I know